overview.md

```diff
@@ -2,17 +2,13 @@ This extension enables you to build and test your MATLAB&reg; project as part of
 
```

Comment:
Thanks for making adjustments to `overview.md`. The changes look good to me. Once you close this pull request, I will open one for additional doc changes. If you cannot address some of the comments in `overview.md`, you can leave them for the upcoming pull request.